erpnext/accounts/report/bank_reconciliation_statement/bank_reconciliation_statement.py

```diff
@@ -95,12 +108,42 @@ def get_columns():
 			"fieldtype": "Date",
 			"width": 110
 		},
+		{
+			"fieldname": "receipt_date",
+			"label": _("Receipt/Voucher Date"),
+			"fieldtype": "Date",
+			"width": 110
+		},
+		{
+			"fieldname": "deposited_date",
+			"label": _("Deposited Date"),
+			"fieldtype": "Date",
+			"width": 110
+		},
+		{
+			"fieldname": "deposited_to",
+			"label": _("Deposited To"),
+			"fieldtype": "Data",
+			"width": 110
+		},
+		{
+			"fieldname": "cheque_status",
```

Comment:
can we use `deposit status`?

The word "cheque" has a spelling problem (its `check` in en-US)